dev/io.openliberty.cdi.4.1.internal.web/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```

Comment:
I know this is a draft PR but ... not obvious to me why this new bundle is needed as there isn't any new code here? If it is needed, then this classpath entry should be Java 17?

Reply:
Java 17 should be set in bnd and other eclipse settings too

Reply:
Thanks @tevans78 on my cleanup list for this PR. The new bundle is needed to have a build dependency specifically on Jakarta Expression Language 6.0. If we don't have this bundle and we update the `osgi.contract` in `dev/io.openliberty.jakarta.expressionLanguage.6.0/bnd.bnd` then the following will occur:

https://github.com/OpenLiberty/open-liberty/pull/27428#issuecomment-2066412990

In this scenario, we need a new bundle that builds the code from the EE10 bundle with a dependency on Jakarta Expression Language 6.0.